expat/CMakeLists.txt

```diff
@@ -124,6 +124,14 @@ if(EXPAT_WITH_LIBBSD)
     endif()
 endif()
 
+if(MSVC)
+    # Minimum supported MSVC version is 1800 = Visual Studio 12.0/2013
+	# See also https://cmake.org/cmake/help/latest/variable/MSVC_VERSION.html
```

Comment:
Can you make both lines indented by four spaces? I'd consider it ready to merge then. Thank you!

Reply:
Ah, sorry. Done.

Reply:
Thank you!